snooty.toml

```diff
@@ -97,3 +97,16 @@ tls = ":abbr:`TLS (Transport Layer Security)`"
 url = ":abbr:`URL (Uniform Resource Locator)`"
 utc = ":abbr:`UTC (Coordinated Universal Time)`"
 vpc = ":abbr:`VPC (Virtual Private Cloud)`"
+
+[[banners]]
+targets = ["command/atlas-dataLakes*.txt"]
+variant = "info"
+value = """\
+    ``atlas dataLake`` commands control the functionality previously \ 
+    named Data Lake, which is now called "Data Federation" in the \ 
+    Atlas UI. The commands will continue to work for Data Federation \ 
+    without interruption. MongoDB's new Data Lake functionality \ 
+    doesn't include CLI. When we introduce a new set of CLI commands \ 
+    for our new Data Lake functionality, we will migrate you to a new \ 
```

Comment:
**Suggestion:** clarify and trim the copy a little. I would also add a link here from "Data Lake" to the docs for the new feature to help them understand the difference
```suggestion
    doesn't include CLI commands. When we introduce CLI commands \ 
    for our new Data Lake functionality, we will migrate you to a new \ 
```